server/src/project/geometry/geometry.service.ts

```diff
@@ -141,6 +141,9 @@ const QUERY_TEMPLATES = {
   'action-types': (queryParamValue) =>
     equalsAnyOf('dcp_name', queryParamValue, 'dcp_projectaction'),
 
+  'zoning-resolutions': (queryParamValue) =>
+    queryParamValue.map(value => `dcp_dcp_project_dcp_projectaction_project/any(o:o/_dcp_zoningresolution_value eq '${value}')`).join(' or '),
```

Comment:
I know it's not obvious but the query syntax here is wrong. It should be XML syntax, not OData. So, this will throw errors for the map.

Reply:
It might be ignoring it because there's a separate "ALLOWED FILTERS" array.

Reply:
@allthesignals  can you explain more about what needs to change?

Reply:
Well, look at the syntax for the other helpers:
![image](https://user-images.githubusercontent.com/5004319/102098299-07148800-3df5-11eb-9f43-896598ae9089.png)

It's fetchXML query syntax. In your code, however, you're using OData Web API syntax.

What needs to change:

1. You'd have to make this code use FetchXML syntax.
2. You'd have to make sure this is actually filtering the map (in your screenshot, it doesn't appear to be filtering the map at all).

I'm OK with this being a separate pull request, but I would remove this line with a TODO.

Let me know!

Reply:
Okay yes I see what you're saying. I'm going to go ahead and merge this and start working on the fetchxml stuff right away, thanks!